lib/fakes.go

```diff
@@ -1,4 +1,4 @@
-package cmd
+package lib
```

Comment:
NIT: Should we keep the name of this file as `fakes_test.go` for clarity?

Reply:
Actually I think this one should remain `fakes.go` and the file that references the fake funcs `lib/preflight_check.go` should be possibly renamed to `lib/preflight_check_test.go`.

Reply:
I renamed `lib/preflight_check.go` to `lib/preflight_check_test.go` to better reflect what's happening as it's just running some ginkgo unit tests.

Reply:
Works for me.